src/db/schema.ts

```diff
@@ -0,0 +1,205 @@
+import {
+  pgTable,
+  foreignKey,
+  uuid,
+  text,
+  timestamp,
+  unique,
+  integer,
+  date,
+  varchar,
+  boolean,
+  pgEnum,
+} from "drizzle-orm/pg-core";
+import { sql } from "drizzle-orm";
+
+export const attendanceStatus = pgEnum("attendance_status", [
+  "asiste",
+  "no_asiste",
+  "por_confirmar",
+  "no_responde",
+]);
+export const bloodType = pgEnum("blood_type", [
+  "a+",
+  "a-",
+  "b+",
+  "b-",
+  "ab+",
+  "ab-",
+  "o+",
+  "o-",
+]);
+export const civilStatus = pgEnum("civil_status", [
+  "soltero",
+  "casado",
+  "divorciado",
+]);
+export const epsProvider = pgEnum("eps_provider", [
+  "aliansalud",
+  "colmedica",
+  "compensar",
+  "sanitas",
+  "sura",
+  "salud_total",
+  "colpatria",
+  "coomeva",
+  "famisanar",
+  "medifiatc",
+  "cafesalud",
+  "susalud",
+  "asmetsalud",
+  "nueva_eps",
+  "sanidad_militar",
+  "sisben",
+]);
+export const eventStatus = pgEnum("event_status", [
+  "por_confirmar",
+  "confirmado",
+  "realizado",
+  "cancelado",
+]);
+export const eventType = pgEnum("event_type", [
```

Comment:
I would look into how many of these enums are actually not going to change in the distant future, updating enum values can be a pain but if all of these are not going to be different soon then this should be fine